content/deploy/cdp_base.adoc

```diff
@@ -0,0 +1,226 @@
+
+However, it is not uncommon to use the target Cloudera-Manager node as the Controller, it is also not uncommon to need to deploy to air-gapped environments or environments where Docker is not permitted.
+
+As such, there is a https://github.com/cloudera-labs/cloudera-deploy/blob/main/centos7-init.sh[script] for deploying the dependencies for Cloudera-Deploy on centos7 as an example in the repo for Power Users in these scenarios. Note that the script covers dependencies for all Cloudera Products on all current Clouds, so you may wish to exclude parts irrelevant to your scenario.
+
+==== Required Definition Files
+There are 2 files that must be present in the definition path for a PvC Base cluster install.
+
+* cluster.yml or definition.yml, or both
+* application.yml
+
+You may optionally supply other files recognised by the Definition parser, such as an inventory template (see Inventories below).
+
+.Cluster.yml
+This file contains the variables that define:
+
+* The CDP Cluster(s) - which services, service configuration parameters, host templates & role allocation, external databases, runtime repositories, security
+* Cloudera Manager - cm repository, management services, CM configs
+* Host configurations - e.g. values set via CM -> All Hosts -> Configuration
+* Auth backends - e.g. ldap/ad
+* Any other vars required for the Plays in-scope (e.g. kdc, haproxy settings)
+
+.Application.yml
+Used to describe additional deployment steps. Although this file must be present, it doesn’t have to do anything and you can simply copy the sample no-op Playbook from the examples/sandbox in Cloudera-Deploy.
+
+.Definition.yml
+Here’s a longer explanation of <<definitions.adoc#_crafting_definitions,Definitions>>
+
+NOTE: It is worth noting that the `admin_password` field from your Profile, as outlined in the Quickstart, will be used as the Cloudera-Manager admin password during cloudera.cluster deployments.
+
+==== Inventories
+An inventory file can be supplied using the traditional -i flag at runtime or by including either
+inventory_static.ini or inventory_template.ini in the definitions path.
+
+.Inventory_static.ini
+An Ansible inventory covering the hosts in-scope for the deployment. An inventory with this name will be picked up automatically if present without using the `-i` to specify the inventory file at runtime.
+
+Common inventory group names are:
+
+* cloudera_manager
+* cluster_master_nodes
+* cluster_worker_nodes
+* cluster_edge_nodes
+
+Each host in these groups should be assigned one of the host templates that is defined in cluster.yml. These can be applied per host by setting `host_template=TemplateName` after the hostname, or per group by setting `host_template=TemplateName` in the group vars. See https://docs.ansible.com/ansible/latest/user_guide/intro_inventory.html#adding-variables-to-inventory[here] for more information on Ansible inventory vars.
```

Comment:
It might be worth including some documentation about the following in this section:

* Hosts can be configured in racks by specifying rack_id=/rackname at the host level
* TLS certificates can be created and deployed to hosts by specifying tls=true at the host level